scripts/import-order-cleanup.py

```diff
@@ -81,7 +81,7 @@ def main():
     for f in go_files:
         parsed, imports_reordered = parse_go_file(f)
         if output == "stdout" and imports_reordered:
-            print f + " imports out of order"
+            print(f + " imports out of order")
```

Comment:
im somehow got python 3 installed on my machine and this is backwards compatible with python 2 anyway